### ISSUE TITLE
consider more time for one-sided communication

### ISSUE THREAD

**Issue by wikfeldt** [ENCCS/intermediate-mpi#60]
if participants of the December 2020 workshop are representative of the intended target audience also for future workshops, we might want to give more time to one-sided communication as that was the topic most were interested in

**Comment by robertodr** [comment]
Yes, I think my original estimates are wildly inaccurate :smile_cat:
